src/mgr/Mgr.cc

```diff
@@ -322,6 +328,31 @@ void Mgr::init()
     "Dump mgr status");
   ceph_assert(r == 0);
 
+#ifdef WITH_LIBCEPHSQLITE
+  dout(4) << "Using sqlite3 version: " << sqlite3_libversion() << dendl;
+  sqlite3_auto_extension((void (*)())sqlite3_cephsqlite_init);
+  {
+    sqlite3* db = nullptr;
+    if (int rc = sqlite3_open_v2("", &db, SQLITE_OPEN_READWRITE, nullptr); rc == SQLITE_OK) {
```

Comment:
where will the temporary database be created by `sqlite3_open_v2()` ?
since there's no mention of a URI with a pool_id or a pool_name, I'm a bit confused

Reply:
if the dbname is `""`" then it opens a memory database. We just need a db handle to be created so that sqlite3 calls into libcephsqlite's initialization function.

Reply:
I've updated the comment with a pointer to libcephsqlite.h.

Reply:
okay; I thought the memory database was created only when the `:memory:` mnemonic was explicitly used in the URI string
maybe it is a good idea to make the memory database creation explicit by specifying the `:memory:` mnemonic

looking at the [man page for `sqlite3_open_v2()` at sqlite.org](https://sqlite.org/c3ref/open.html), it says, **If the filename is an empty string, then a private, temporary on-disk database will be created.**

so maybe, the man page and the actual implementation is different

Reply:
Hmm, that may be right. I'll just switch to `:memory:`. Thanks.